lib/content-services/src/lib/search/components/search-filter-chips/search-facet-chip/search-facet-chip.component.html

```diff
@@ -2,6 +2,7 @@
           disableRipple
           class="adf-search-filter-chip"
           [class.adf-search-toggle-chip]="(facetField.displayValue$ | async) || menuTrigger.menuOpen"
+          [disabled]="!isPopulated()"
```

Comment:
We should not call functions in html. We have raised jiras to replace all this kind of places. @swapnil-verma-gl was already fixing this in one of our repositories.